build-tests/api-extractor-scenarios/etc/test-outputs/dynamicImportType3/api-extractor-scenarios.api.md

```diff
@@ -0,0 +1,19 @@
+## API Report File for "api-extractor-scenarios"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Lib1GenericType } from 'api-extractor-lib1-test';
+import { Lib1Interface } from 'api-extractor-lib1-test';
+
+// @public (undocumented)
+export interface IExample {
+    // (undocumented)
+    generic: Lib1GenericType.Lib1GenericType;
```

Comment:
@javier-garcia-meteologica this is a simplified repro for https://github.com/microsoft/rushstack/issues/1754

Do you think it's much work to handle in this PR?  If not, we could solve it as a separate PR.

Reply:
I will try to fix it

Reply:
Done

Reply:
Now it's done and working